### ISSUE TITLE
Discussion

### ISSUE THREAD

**Issue by tfsojon** [intelops/external-docs#4]
@chandu-intelops Please commit and push the changes you have made otherwise my changes will conflict with yours.

**Comment by devopstoday11** [comment]
@tfsojon - I am not touching anything in this repo. please proceed and push your changes. The changes which you have seen in the video I provided were the ones I am playing with and it is a copy for my internal purpose. Please push your changes, I am waiting on you.

**Comment by tfsojon** [comment]
@devopstoday11, Not able to push code. Please add @tfsojon as a maintainer in this repo.

**Comment by devopstoday11** [comment]
@tfsojon 
Last time you submitted PR. Please follow same process. It's a public repo. When you fork it and do the work, you can submit PR.

**Comment by devopstoday11** [comment]
@tfsojon 
What does TOC mean?

**Comment by tfsojon** [comment]
Table of Content

**Comment by devopstoday11** [comment]
@tfsojon 
The documentation versioning is still missing.

**Comment by devopstoday11** [comment]
@tfsojon ,
Different topic.....
If I ever want to hide any page or any block or row or a particular section in a page from main site , how to hide them, where do I declare that hide & show parameters? I don't want to depend on Forestry. The less we depend on Forestry, the better. 
Please make a video a show how to achieve the above scenario.

**Comment by tfsojon** [comment]
fix search result issue and added random features and fixes

**Comment by devopstoday11** [comment]
@tfsojon  you made the search work for the latest and also old versions, right?

**Comment by devopstoday11** [comment]
@tfsojon ,
new feedback/issues - https://www.loom.com/share/d87e953b601c44beabda25b8370a9de5

**Comment by tfsojon** [comment]
@devopstoday11 Are you available now?

**Comment by devopstoday11** [comment]
@tfsojon 
What's up?

**Comment by tfsojon** [comment]
![image](https://user-images.githubusercontent.com/87603786/229267593-231c8cb4-3ec5-4968-b8a7-c0a7f446e964.png)

**Comment by tfsojon** [comment]
check the new PR https://github.com/intelops/external-docs/pull/20

**Comment by tfsojon** [comment]
and the 404 error showing because the main branch and chandu branch's content is not same.

**Comment by devopstoday11** [comment]
> ![image](https://user-images.githubusercontent.com/87603786/229267593-231c8cb4-3ec5-4968-b8a7-c0a7f446e964.png)
> 
> 

I was not able to understand how to use that structure. Please explain in a video @tfsojon

**Comment by devopstoday11** [comment]
> and the 404 error showing because the main branch and chandu branch's content is not same.

I thought it is same. Let me check again and get back to you @tfsojon

**Comment by tfsojon** [comment]
Not same. Please check the PR.

**Comment by tfsojon** [comment]
> I was not able to understand how to use that structure. Please explain in a video @tfsojon

What will be the content structure of your changelog? that's what I asked for.

**Comment by devopstoday11** [comment]
> > and the 404 error showing because the main branch and chandu branch's content is not same.
> 
> I thought it is same. Let me check again and get back to you @tfsojon

404 error which I mentioned in the video regarding "edit in git repo" button is for chandu branch only. I didn't push to main branch yet. So, the issue is still there in chandu branch. after you fix it, then I will merge to main branch. @tfsojon

**Comment by tfsojon** [comment]
we have to set the git url here in config.toml file. currently the url branch is `main` - 
![image](https://user-images.githubusercontent.com/87603786/229268995-95ab1def-c85a-4e33-bc8e-404057099e23.png)

also did you checked the new PR? specially the changelog content folder?
What will be the content structure of your changelog?
Is this correct? - https://github.com/tfsojon/external-docs/tree/main/content/1.0.0/changelog

**Comment by devopstoday11** [comment]
> > > and the 404 error showing because the main branch and chandu branch's content is not same.
> > 
> > 
> > I thought it is same. Let me check again and get back to you @tfsojon
> 
> 404 error which I mentioned in the video regarding "edit in git repo" button is for chandu branch only. I didn't push to main branch yet. So, the issue is still there in chandu branch. after you fix it, then I will merge to main branch. @tfsojon

@tfsojon 
one more thing which I explained in video was, when I click on "edit in git repo"  button, no need to open the exact file in the github. Just make it open the folder or just the git repo. That's enough. 

For example, when I click on edit button when I am inside documentation of a topic, then it should open 
https://github.com/intelops/external-docs/tree/main/content/1.0.0/compage 
If I am inside change log and I click edit button it should take me to the changelog folder - 
https://github.com/intelops/external-docs/tree/main/content/1.0.0/changelog 

So the syntax would be like this -
![image](https://user-images.githubusercontent.com/59978378/229269417-07d33a8a-258e-4cc4-8386-7c4c59dbbcd2.png)


Hope you got what needs to be opened when the edit button is clicked? @tfsojon

**Comment by devopstoday11** [comment]
@tfsojon can you join a quick call to discuss on changelog usage. I think the folder structure you gave for changelog is fine but I am not able to understand how to use it. 
I can share google meet link and we can discuss quickly for 5 minutes

**Comment by tfsojon** [comment]
ok send me the link please

**Comment by devopstoday11** [comment]
> ok send me the link please

please join https://meet.google.com/vrb-xbjo-ztc 
@tfsojon

**Comment by devopstoday11** [comment]
Thanks for the quick call, @tfsojon. Please proceed with the items discussed.
